components/camel-huaweicloud-smn/src/main/java/org/apache/camel/component/huaweicloud/smn/constants/SmnProperties.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.camel.component.huaweicloud.smn.constants;
+
+public class SmnProperties {
+
+    public SmnProperties() {
+    }
+
+    // request properties
+    public static String TEMPLATE_NAME = "CamelHwCloudSmnTemplateName";
```

Comment:
immutable `final` is missing from the constants

Reply:
good catch :) just checked in the fix. thanks